pkg/kubectl/rollback.go

```diff
@@ -125,16 +137,16 @@ func (r *DeploymentRollbacker) Rollback(obj runtime.Object, updatedAnnotations m
 	result := ""
 
 	// Get current events
-	events, err := r.c.CoreV1().Events(d.Namespace).List(metav1.ListOptions{})
+	events, err := r.c.CoreV1().Events(namespace).List(metav1.ListOptions{})
 	if err != nil {
 		return result, err
 	}
 	// Do the rollback
-	if err := r.c.ExtensionsV1beta1().Deployments(d.Namespace).Rollback(deploymentRollback); err != nil {
+	if err := r.c.ExtensionsV1beta1().Deployments(namespace).Rollback(deploymentRollback); err != nil {
```

Comment:
We should switch to `AppsV1` since extensions is deprecated.

Reply:
AppsV1 does not have the method "Rollback". Deployment rollback is implemented very differently from DaemonSets and StatefulSets for some reason. I'm going to add a TODO to update the entire Deployment rollback process. But for now, I think we should keep this change as small as possible.